tests/integration/targets/zabbix_group_info/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: test - create new Zabbix group
+  zabbix_group:
+    server_url: "{{ zabbix_server_url }}"
+    login_user: "{{ zabbix_login_user }}"
+    login_password: "{{ zabbix_login_password }}"
+    host_groups:
+      - zbxgrp_example_group01
+      - zbxgrp_example_group02
+    state: present
+  register: zbxgrp_new
+
+- name: assert that group was created
+  assert:
+    that: zbxgrp_new is changed
+
+- name: test - get one hostgroup info
+  zabbix_group_info:
+    server_url: "{{ zabbix_server_url }}"
+    login_user: "{{ zabbix_login_user }}"
+    login_password: "{{ zabbix_login_password }}"
+    hostgroup_name:
+      - zbxgrp_example_group01
+  register: get_hostgorup_info_result
+
+- debug: var=get_hostgorup_info_result
```

Comment:
forgotten debug?

Reply:
Thank you @D3DeFi for pointed out.
I have forgotten to remove the debug line so that I will remove the debug line.